Compatibility/com.yayo.lostTechCore.2/1.4/Defs/Adventures_in_Fantasy_World.xml

```diff
@@ -0,0 +1,75 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+  <ScenarioDef>
+    <defName>AdventuresinFantasyWorld</defName>
+    <label>Adventures in Fantasy World</label>
+	<description>Three colonists set out from their cozy home to find adventure out in the world! No crafting or research allowed, everything you use or earn must be gathered from out there in the wide world!</description>
+	<scenario>
+		<summary>Three colonists, starting a cozy inn, set out into the world to find wealth and rewards beyond measure.</summary>
+		<playerFaction>
+			<def>PlayerFaction</def>
+			<factionDef>PlayerLostTechnology</factionDef>
```

Comment:
If you're feeling daring, perhaps defaulting this to a non losttech faction then just having a patch to swap it to lost tech if present so that players not wanting lost tech can still use it?